code/__DEFINES/medals.dm

```diff
@@ -0,0 +1,24 @@
+//This file holds all the medals in the game. Adding another one is as simple as just editing the define.
+
+//These medals are mundane or hold no real significance.
+#define TEST_MEDAL "Hello World!" //Unachieveable.
+
+#define MEDAL_NEW_PLAYER "Fresh Meat" //Log in for the first time.
+#define MEDAL_FIRST_DEATH "Remember This Moment" //Die for the first time.
+
+//These medals are feats of strength, awarded through difficult tasks or incredible endurance.
+#define MEDAL_LEGION "The Necropolis Awaits" //Kill Legion.
+#define MEDAL_BUBBLEGUM "All out of Bubblegum" //Kill Bubblegum.
```

Comment:
I'm not completely sure actually. Unlike the scoreboard the page does seem to have an option for renaming medals.
I'll run some tests.